siliconcompiler/schema/schema_cfg.py

```diff
@@ -28,7 +28,8 @@ def scparam(cfg,
             switch=None,
             example=None,
             schelp=None,
-            enum=None):
+            enum=None,
+            pernode='none'):
```

Comment:
What do you think about 'never' or 'forbidden' as the opposite option to 'mandatory', instead of 'none'?